### PR TITLE
bugfix; Update status when configmap/secret failed to merge initially

### PR DESCRIPTION
**Pull request by tomahawk28**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: service/controller/app/resource/status/create.go [hunk 1]
```diff
@@ -37,26 +37,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
...
-		return nil
-	} else if err != nil {
-		return microerror.Mask(err)
-	}
-
-	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("found status for chart %#q in namespace %#q", cr.Name, r.chartNamespace))
-
-	chartStatus := key.ChartStatus(*chart)
 	var desiredStatus v1alpha1.AppStatus
 	if cc.Status.ChartStatus.Status != "" {
```
Comment (tomahawk28) [inline service/controller/app/resource/status/create.go:41]: Check `context` first would prevent to find chart CR in tenant clusters.

Comment (rossf7) [reply service/controller/app/resource/status/create.go:41]: Minor but please add a newline before the if for readability.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Fixed
+
+- Update the status when failed to merge configMaps/secrets on the initial reconciliation.
```
Comment (rossf7) [inline CHANGELOG.md:12]:
```suggestion
- Update the status when failing to merge configMaps or secrets on the initial reconciliation.
```

**Review by rossf7 — APPROVED** [review]
LGTM 🚀

**Pull request merged**